packages/docs-examples/components/content-panel/content-panel-with-grid/content-panel-with-grid-example.ts

```diff
@@ -0,0 +1,320 @@
+                            </button>
+                            <button [color]="componentColors.ContrastFade" kbq-button>
+                                <i kbq-icon="kbq-square-multiple-o_16"></i>
+                                Copy
+                            </button>
+                            <button [color]="componentColors.ContrastFade" kbq-button>
+                                <i kbq-icon="kbq-trash_16"></i>
+                                Delete
+                            </button>
+                        </div>
+                    </kbq-content-panel-header>
+                    <kbq-content-panel-body class="example-content-panel-body">
+                        @for (_i of [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]; track $index) {
+                            <p>
+                                In computing [{{ $index }}], a denial-of-service attack (DoS attack) is a cyber-attack
+                                in which the perpetrator seeks to make a machine or network resource unavailable to its
+                                intended users by temporarily or indefinitely disrupting services of a host connected to
+                                a network. Denial of service is typically accomplished by flooding the targeted machine
+                                or resource with superfluous requests in an attempt to overload systems and prevent some
+                                or all legitimate requests from being fulfilled. The range of attacks varies widely,
+                                spanning from inundating a server with millions of requests to slow its performance,
+                                overwhelming a server with a substantial amount of invalid data, to submitting requests
+                                with an illegitimate IP address.
+                            </p>
+                        }
+                    </kbq-content-panel-body>
+                    <kbq-content-panel-footer>
+                        <button [color]="componentColors.Contrast" kbq-button>Button 0</button>
+                        <button [color]="componentColors.ContrastFade" kbq-button>Button 1</button>
+                        <button [color]="componentColors.ContrastFade" kbq-button>Button 2</button>
+                    </kbq-content-panel-footer>
+                </kbq-content-panel>
+            </kbq-content-panel-container>
+        </kbq-modal-body>
+    `,
+    styles: `
+        :host {
+            display: flex;
+            flex-direction: column;
+            height: 90vh;
+        }
+
+        .example-modal-body {
+            height: 100%;
+            max-height: unset;
+        }
+
+        .example-modal-body:has(.kbq-content-panel-container__opened) {
+            padding-right: 0;
+        }
+
+        ::ng-deep .example-modal-title {
+            padding: var(--kbq-size-xl) var(--kbq-size-xxl);
+        }
+
+        ::ng-deep .example-modal-title .kbq-modal-title {
```

Comment:
судя по коду, никогда не удовлетворится, должно быть 


```suggestion
        ::ng-deep .example-modal-title.kbq-modal-title {
```

Reply:
<img width="390" height="54" alt="image" src="https://github.com/user-attachments/assets/02622bf0-3ff5-4ad2-841e-f9e2f15a8213" />
<img width="443" height="176" alt="image" src="https://github.com/user-attachments/assets/c621e982-a579-4206-8737-627fe748fb39" />